.buildkite/filebeat/filebeat-pipeline.yml

```
@@ -111,6 +111,38 @@ steps:
         artifact_paths:
           - "filebeat/build/*.xml"
           - "filebeat/build/*.json"
+          - "filebeat/build/integration-tests/*"
+          - "filebeat/build/integration-tests/Test*/*"
+          - "filebeat/build/integration-tests/Test*/data/**/*"
+        plugins:
+          - test-collector#v1.10.2:
+              files: "filebeat/build/TEST-*.xml"
+              format: "junit"
+              branches: "main"
+              debug: true
+        notify:
+          - github_commit_status:
+              context: "filebeat: Go Integration Tests"
+
+      - label: ":ubuntu: Filebeat: Integration tests as Root"
+        command: |
+          sudo usermod -a -G systemd-journal $$USER
```

if i am not wrong, this is how bk escapes the env vars.

```suggestion
          sudo usermod -a -G systemd-journal \$USER
```

---

Both are supported according to the [docs](https://buildkite.com/docs/pipelines/environment-variables#runtime-variable-interpolation).

`$$USER` worked in the last run, then it failed because I didn't compile the test binary :man_facepalming: .